dashboards/euc/euc-dashboard.yaml

```
@@ -0,0 +1,657 @@
+          Alias: euc_dashboard_combined
+          Source:
+            PhysicalTableId: c00dd88a-e0c2-498c-b7e0-0a4e1053a4ed
+        b9e0e048-8364-471f-b2bf-935acbd46036:
+          Alias: Intermediate Table
+          DataTransforms:
+          - TagColumnOperation:
+              ColumnName: region
+              Tags:
+              - ColumnGeographicRole: STATE
+          - ProjectOperation:
+              ProjectedColumns:
+              - workspaceid
+              - username
+              - accountid
+              - computetype
+              - directoryid
+              - directoryname
+              - directoryalias
+              - directorytype
+              - directorymaintenance
+              - state
+              - connectionstatus
+              - lastconnected
+              - lastinventoryrun
+              - runningmode
+              - operatingsystem
+              - protocol
+              - computername
+              - ipaddress
+              - year
+              - month
+              - billing_period
+              - usage_date
+              - payer_account_id
+              - linked_account_id
+              - invoice_id
+              - charge_type
+              - line_item_product_code
+              - line_item_usage_type
+              - line_item_operation
+              - line_item_line_item_description
+              - line_item_resource_id
+              - line_item_usage_amount
+              - line_item_unblended_cost
+              - bill_billing_entity
+              - service
+              - pricing_public_on_demand_cost
+              - pricing_term
+              - pricing_unit
+              - product_product_family
+              - product_instance_type
+              - product_instance_family
+              - region
+              - product_product_name
+              - product_operating_system
+              - product_group
+              - product_bundle_description
+              - product_bundle_group
+              - product_resource_type
+              - product_storage
+              - product_running_mode
+              - product_group_description
+              - product_software_included
+              - resource_id
+              - resource_type
+              - resource_directory_id
+              - usage_quantity
+              - unblended_cost
+              - amortized_cost
+              - account_id
+              - account_name
+          Source:
+            JoinInstruction:
+              LeftOperand: 9cff1849-cd58-40d4-9f6c-689073f9df37
+              RightOperand: 851015ef-7f8c-43ea-aa89-5318b5848aac
+              Type: LEFT
+              OnClause: '{linked_account_id} = {account_id}'
+      ImportMode: SPICE
+    dependsOn:
+      views:
+      - account_map
+      - euc_dashboard_combined
+    schedules:
+    - default
+crawlers: {}
+views:
+  euc_dashboard_metrics:
+    data: |-
+      CREATE OR REPLACE VIEW "${athena_database_name}".euc_dashboard_metrics AS
```

```suggestion
      CREATE VIEW "${athena_database_name}".euc_dashboard_metrics AS
```

This will only create this view but will NOT update it on recursive update if customer adjusted this

---

Good point.. will change